tutor/templates/config/defaults.yml

```diff
@@ -38,8 +38,9 @@ K8S_NAMESPACE: "openedx"
 LANGUAGE_CODE: "en"
 LMS_HOST: "www.myopenedx.com"
 LOCAL_PROJECT_NAME: "{{ TUTOR_APP }}_local"
+MEILISEARCH_HOST: "meilisearch.{{ LMS_HOST }}"
 MEILISEARCH_URL: "http://meilisearch:7700"
-MEILISEARCH_PUBLIC_URL: "{% if ENABLE_HTTPS %}https{% else %}http{% endif %}://meilisearch.{{ LMS_HOST }}"
+MEILISEARCH_PUBLIC_URL: "{% if ENABLE_HTTPS %}https{% else %}http{% endif %}://{{ MEILISEARCH_HOST }}"
```

Comment:
Why is MEILISEARCH_PUBLIC_URL not usable? Why do we need the host explicitly?

Reply:
Great questions, and thank you for the review. If we already have a convention to use LMS_HOST, PREVIEW_HOST, etc., it only makes sense to have it as MEILISEARCH_HOST.

Adding to that MEILISEARCH_HOST helps us to configure the host URL, MEILISEARCH_PUBLIC_URL does the same but one has to be aware of the protocol which again was not a convention that was followed. Hence it made more sense to rename it and keep the MEILISEARCH_PUBLIC_URL unchanged.

Let me know  if it makes sense 😄

Reply:
Good enough. 
Why is this being merged to main and not release? It does not seem to be a breaking change.

Reply:
@DawoudSheraz  sorry I wasn't aware of the contribution norms I have read those again and have made the changes.